packages/cascara/src/modules/DataDate/DataDate.js

```diff
@@ -0,0 +1,74 @@
+import React, { useContext } from 'react';
+import { Input } from 'reakit/Input';
+import pt from 'prop-types';
+import { ModuleContext } from '../context';
+import styles from '../DataModule.module.scss';
+
+import ModuleErrorBoundary from '../ModuleErrorBoundary';
+
+const propTypes = {
+  /** A module can have an Attribute, which will be used as form field name */
+  attribute: pt.string,
+  /** A Module can be defined to not present an editing state */
+  isEditable: pt.bool,
+  /** Presents the input without a label. NOT USER CONFIGURABLE */
+  isLabeled: pt.bool,
+  /** A Module needs to have a unique label relative to its context */
+  label: pt.string,
+  max: pt.string,
+  min: pt.string,
+  /** A Module can have a value */
+  value: pt.string,
+};
+
+const DataDate = ({
+  attribute,
+  isEditable = true,
+  isLabeled = true,
+  label,
+  max,
+  min,
```

Comment:
We removed `min` and `max` properties below on the input... but we are still destructuring them here on the DataDate function. That is effectively muting them from being passed to the Input component with `{...rest}`. We should remove them here so we do not prevent them from passing down.